src/features/click-to-play.js

```diff
@@ -1346,6 +1336,8 @@ const updateHandlers = {
     }
 }
 
+const knownMessageResponseType = Object.prototype.hasOwnProperty.bind(messageResponseHandlers)
```

Comment:
This is to avoid response messages being able to trigger functions that were inherited from Object.prototype to be called.